config/checkstyle-examples-suppressions.xml

```diff
@@ -34,6 +34,9 @@
   <!-- Needs more compact view of the file -->
   <suppress checks="Indentation"
     files="returncount[\\/]Example\d+.java"/>
+  <!-- Needs more compact view of the file -->
+  <suppress checks="Indentation"
+    files="singlelinejavadoc[\\/]Example\d+.java"/>
```

Comment:
Please follow this validation, and it will be compact.
Indentation should be +2, you have +4from class to method

Reply:
Done